src/components/inputCo.js

```diff
@@ -96,12 +76,14 @@ const InputCo = ({
         onChange={setValue}
         error={errors !== ""}
       />
-      {helperText && (
-        <FormHelperText className={classes.helperText} error={errors !== ""}>
-          <span>{errors && component}</span>
-          <span>{(!hasErrors && helperText) || errors}</span>
-        </FormHelperText>
-      )}
+      <div className={classes.helperTextContainer}>
+        {helperText && (
+          <FormHelperText className={classes.helperText} error={errors !== ""}>
```

Comment:
the `errors !== ""` is repeating, could you extract it out in a constant like `hasError`?

Reply:
Done